cmd/cloudFoundryDeleteService.go

```diff
@@ -30,10 +37,54 @@ func cloudFoundryDeleteService(CloudFoundryDeleteServiceOptions cloudFoundryDele
 	return nil
 }
 
+func cloudFoundryDeleteServiceKeys(CloudFoundryDeleteServiceOptions cloudFoundryDeleteServiceOptions, c execRunner) error {
```

Comment:
I would name the parameter just `options`. Capital spelling also has special meaning in Go, which is why it seems strange in addition to the long name.

Reply:
Changed it accordingly for all functions. It's now using only `options` parameter.